src/bin/purge_ttl.rs

```diff
@@ -17,8 +17,15 @@ use grpcio::{CallOption, ChannelBuilder, ChannelCredentials, EnvBuilder, Metadat
 use log::{info, trace, warn};
 use url::{Host, Url};
 
+const BATCH_SIZE: i64 = 1000;
```

Comment:
nit: Insert: "If it's a static, it should be an CLI option" spiel here.

Reply:
Yeah, I should make it an env var.

Reply:
Done.